apps/native-component-list/src/screens/InAppPurchases/README.md

```diff
@@ -0,0 +1,16 @@
+# Expo Payments
+
+This is an ejected Expo app that can be used to test the `expo-in-app-purchases` module.
+
+It has a bundle ID of `dev.expo.Payments` and an application ID of `com.expo.payments`.
```

Comment:
`bare-expo` uses an application ID of `dev.expo.payments`. Do you think we should change the respective configuration in the Google Play Console?

Reply:
Might make sense to just create a few more payments.